### DISCUSSION TITLE
Explore: Hard to compare 2+ values in graph in Explore

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#61621] [Explore]
Blocked by https://github.com/grafana/grafana/issues/61615

Currently Explore's graph tooltip is showing only 1 value in tooltip (the hovered-over one) which makes it hard/impossible to compare multiple values in time. 

<img width="1909" alt="image" src="https://user-images.githubusercontent.com/30407135/212899383-cc6e1230-8ef7-45ff-b066-b82004d06fd8.png">

The possible solution could be using `multi` tooltip that shows multiple values, but in current implementation, tooltip blocks large part of graph and is in the way. 

<img width="814" alt="image" src="https://user-images.githubusercontent.com/30407135/212900576-39b1caa9-ad78-4f4f-8e17-29189e34d9b8.png">

**Environment**:
- Grafana version: 9.3.0

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82813. Feel free to continue the discussion around this there. Thank you!
